spec.bs

```diff
@@ -1224,9 +1223,23 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     1. [=map/For each=] joiningOrigin → |groups| of |perSignalsUrlGenerator|:
       1. [=list/For each=] |ig| of |groups|:
         1. If |ig|'s [=interest group/bidding url=] is null, [=iteration/continue=].
+        1. Let |directFromSellerSignals| be null.
+        1. Let |buyerDirectFromSellerSignals| be {{undefined}}.

```

Comment:
Why {{undefined}} instead of null? Isn't this an infra type, not a WebIDL type?

Reply:
sorry, should be clear that this is part of the todo of converting the signal to JS type. I had a mix of both, and need to fix that.

Reply:
changed to null, but still need to do the conversion.